src/compute-client/src/command.proto

```diff
@@ -41,7 +41,7 @@ message ProtoComputeCommand {
         ProtoInstanceConfig create_instance = 1;
         google.protobuf.Empty drop_instance = 2;
         ProtoCreateDataflows create_dataflows = 3;
-        mz_storage.client.ProtoAllowCompaction allow_compaction = 4;
+        mz_storage.protocol.client.ProtoAllowCompaction allow_compaction = 4;
```

Comment:
This will probably break backwards compat? Probably all the protobuf changes?

Reply:
I think protobuf only cares about the shape of the data and not how the fields are named or how the packages are organized.